gocd/generated-pipelines/relay-next-monitor.yaml

```diff
@@ -0,0 +1,109 @@
+format_version: 10
+pipelines:
+  deploy-relay-next-monitor:
+    display_order: 2
+    environment_variables:
+      SENTRY_REGION: monitor
+    group: relay-next
+    lock_behavior: unlockWhenFinished
+    materials:
+      deploy-relay-next-us-pipeline-complete:
+        pipeline: deploy-relay-next-us
+        stage: pipeline-complete
+      relay_repo:
+        branch: master
+        destination: relay
+        git: git@github.com:getsentry/relay.git
+        shallow_clone: true
+    stages:
+      - checks:
+          fetch_materials: true
+          jobs:
+            checks:
+              elastic_profile_id: relay
+              environment_variables:
+                GITHUB_TOKEN: '{{SECRET:[devinfra-github][token]}}'
+              tasks:
+                - script: |
+                    ##!/bin/bash
+
+                    /devinfra/scripts/checks/githubactions/checkruns.py \
+                        getsentry/relay \
+                        "${GO_REVISION_RELAY_REPO}" \
+                        "Integration Tests" \
+                        "Test (macos-latest)" \
+                        "Test (windows-latest)" \
+                        "Test All Features (ubuntu-latest)" \
+                        "Push GCR Docker Image"
+              timeout: 1800
+      - deploy-production:
+          fetch_materials: true
+          jobs:
+            create_sentry_release:
+              elastic_profile_id: relay
+              environment_variables:
+                SENTRY_AUTH_TOKEN: '{{SECRET:[devinfra-temp][relay_sentry_auth_token]}}'
+                SENTRY_ORG: sentry
+                SENTRY_PROJECT: relay
+                SENTRY_URL: https://sentry.my.sentry.io/
+              tasks:
+                - script: |
+                    ##!/bin/bash
+
+                    ./relay/scripts/create-sentry-release "${GO_REVISION_RELAY_REPO}"
+              timeout: 1200
+            deploy:
+              elastic_profile_id: relay
+              tasks:
+                - script: |
+                    ##!/bin/bash
+
+                    eval $(/devinfra/scripts/regions/project_env_vars.py --region="${SENTRY_REGION}")
+
+                    /devinfra/scripts/k8s/k8stunnel
+
+                    /devinfra/scripts/k8s/k8s-deploy.py \
+                        --label-selector="service=relay,deploy_if_production=true" \
+                        --image="us.gcr.io/sentryio/relay:${GO_REVISION_RELAY_REPO}" \
+                        --container-name="relay"
+              timeout: 1200
+      - progress-to-pops:
+          approval:
+            allow_only_on_success: true
+            type: manual
```

Comment:
Will this pause the deployment pipeline until manual confirmation?

Reply:
Sorry I missed this comment - Yes it will, so the process for now will be:

1. Manually start deploy-relay
2. US relay is deployed
3. Manually start deploy-pops for US
4. US relay pops is deployed
5. s4s relay is deployed
6. Manually start deploy-pops for s4s
7. s4s relay pops is deployed